demo.py

```diff
@@ -23,24 +23,29 @@ def main():
   # Define the supply chain
   #############################################################################
   """
+  # Create keys
+  print "Generate keypair for Alice..."
```

Comment:
I understand this is demo code, so I'm not completely worried about this, but we should use the `print()` function rather than the `print` statement.